include/aikido/planner/vectorfield/VectorFieldPlanner.hpp

```diff
@@ -0,0 +1,87 @@
+#ifndef AIKIDO_PLANNER_VECTOR_FIELD_PLANNER_HPP_
+#define AIKIDO_PLANNER_VECTOR_FIELD_PLANNER_HPP_
+
+#include <boost/function.hpp>
+#include <aikido/constraint/Testable.hpp>
+#include <aikido/statespace/dart/MetaSkeletonStateSpace.hpp>
+#include <aikido/trajectory/Spline.hpp>
+
+namespace aikido {
+namespace planner {
+namespace vectorfield {
+
+struct VectorFieldPlannerStatus
+{
+  enum Enum
+  {
+    TERMINATE,
+    CACHE_AND_TERMINATE,
+    CACHE_AND_CONTINUE,
+    CONTINUE
+  };
+};
+
+/// Callback function of joint velocity calculated by a vector field and a
+/// MetaSkeleton.
+/// \param stateSpace MetaSkeleton state space
+/// \param t planned time of a given duration
```

Comment:
Nit: I think the convention is beginning with a capital letter as `Planned time ...`. There are many docstring begin with a lower letter.